ocs_ci/ocs/constants.py

```diff
@@ -464,3 +464,7 @@
 REPO_MAPPING = {
     '4.2.0': OCP4_2_REPO
 }
+
+# Cluster name limits
+CLUSTER_NAME_MIN_CHARACTERS = 5
```

Comment:
@gklein ,

Any reason behind min char length limit ?

Reply:
I think we need to use some minimal text to identify each cluster and to help prevent duplicate names.

I don't mind to reduce it, but I think it would be risky to allow 1 or 2 character long cluster names.

Reply:
hmm makes sense.